### ISSUE TITLE
Site not being refreshed after added property

### ISSUE THREAD

**Issue by Alfareiza** [SteinOveHelset/djangobnb#2]
When the property is added using the modal, the property is in fact added so we can check it in the django admin perfectly, however this property is not shown in the front.

In the [AddPropertyModal](https://github.com/SteinOveHelset/djangobnb/blob/bdc3b56430959a77c135d04d8647197ae5fbe7ba/app/components/modals/AddPropertyModal.tsx#L84) the router.push goes the user to the home but it doesn't request all the properties to the API and show them in the front.

So, once the user add the property, I have to refresh the site.

I know this repo has academic purposes. I'm just making the behavior clear whit this issue. Thanks by the way.

**Comment by steakmeatdev** [comment]
Yeah, I noticed this issue too. The reason is that the auth tokens that the app receives from the backend are set as HttpOnly (true) and this causes a problem when sending mutating requests on client side, as it can't get any tokens and hence it can't perform some operations to the backend properly. Setting SameSite to Strict or Lax is also important in my opinion to prevent CSRF attacks.

**Comment by SteinOveHelset** [comment]
Hey, thanks for pointing this out. Will hopefully help someone else :-)
